python/paddle/trainer/config_parser.py

```diff
@@ -2457,13 +2457,11 @@ def __init__(
                 input_layer = self.get_input_layer(input_index)
                 operator_conf.input_sizes.append(input_layer.size)
                 operator_input_index.append(input_index)
-            if self.config.size == 0:
-                size = operator.calc_output_size(operator_conf.input_sizes)
-                if size != 0:
+            size = operator.calc_output_size(operator_conf.input_sizes)
```

Comment:
In current version, as line 2461 is the same as line 2465 (`size = operator.calc_output_size(operator_conf.input_sizes)`), thus no matter line 2460 (`if self.config.size == 0`) is true, line 2461 will be executed. So I move this line up front.